config/software/chef.rb

```diff
@@ -71,7 +71,8 @@
   # use the rake install task to build/install chef-config/chef-utils
   command "rake install:local", env: env
 
-  gemspec_name = windows? ? "chef-universal-mingw32.gemspec" : "chef.gemspec"
+  # gemspec_name = windows? ? "chef-universal-mingw32.gemspec" : "chef.gemspec"
+  gemspec_name = "chef.gemspec"
```

Comment:
Would this start to fail on the windows 17 and 16 builds?

Reply:
That definitely needs a refactor

Reply:
Solved